src/conda_package_handling/tarball.py

```diff
@@ -81,6 +82,11 @@ def create(prefix, file_list, out_fn, out_folder=os.getcwd(), **kw):
             ".tar.bz2",
             "bzip2",
         )
+
+        with open(out_file, "rb") as f:
+            sha256 = hashlib.sha256(f.read()).hexdigest()
+            LOG.info("Created '%s' with sha256 '%s'", out_file, sha256)
+
```

Comment:
Same as above. This might be worth its own utility function. 

[Something like this](https://github.com/conda-incubator/condacolab/blob/b9393b49e1806df8397038d1b2aaddc632c22bc3/condacolab.py#L39-L44).